tests/test-p11test.sh

```diff
@@ -11,8 +11,10 @@ if [[ ! -f $P11LIB ]]; then
     exit 77;
 fi
 
-# The Ubuntu has old softhsm version not supporting this feature
-grep "Ubuntu 18.04" /etc/issue && echo "WARNING: Not supported on Ubuntu 18.04" && exit 77
+if [ "${TOKENTYPE}" == "softhsm" ]; then
+    # The Ubuntu has old softhsm version not supporting this feature
+    grep "Ubuntu 18.04" /etc/issue && echo "WARNING: Not supported on Ubuntu 18.04" && exit 77
+fi
```

Comment:
not sure if this is relevant anymore. Ubuntu 18 is EOL (except for Extended support), but certainly for the CI use so I would just drop this condition.